src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -77,6 +80,34 @@ public static function create($file, CacheItemPoolInterface $fallbackPool)
         return $fallbackPool;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function get(string $key, callable $callback, float $beta = null)
+    {
+        if (!\is_string($key)) {
```

Comment:
Just asking: above we have `string $key` in the method argument ... will this `if (!is_string($key))` code be ever executed if we pass a non-string argument?